content/en/docs/tasks/debug-application-cluster/debug-cluster.md

```diff
@@ -36,28 +36,28 @@ of the relevant log files.  (note that on systemd-based systems, you may need to
 
 ### Master
 
-   * /var/log/kube-apiserver.log - API Server, responsible for serving the API
-   * /var/log/kube-scheduler.log - Scheduler, responsible for making scheduling decisions
-   * /var/log/kube-controller-manager.log - Controller that manages replication controllers
+   * `/var/log/kube-apiserver.log` - API Server, responsible for serving the API
+   * `/var/log/kube-scheduler.log` - Scheduler, responsible for making scheduling decisions
+   * `/var/log/kube-controller-manager.log` - Controller that manages replication controllers
 
 ### Worker Nodes
 
-   * /var/log/kubelet.log - Kubelet, responsible for running containers on the node
-   * /var/log/kube-proxy.log - Kube Proxy, responsible for service load balancing
+   * `/var/log/kubelet.log` - Kubelet, responsible for running containers on the node
+   * `/var/log/kube-proxy.log` - Kube Proxy, responsible for service load balancing
 
 ## A general overview of cluster failure modes
 
 This is an incomplete list of things that could go wrong, and how to adjust your cluster setup to mitigate the problems.
 
-Root causes:
+##### Root causes:
```

Comment:
An H5 seems very specific. Was there a reason to go this deep?

Reply:
rational was to get to articulate the 3 pillars: `Root causes`, `Specific scenarios` & `Mitigations` from the rest of the actions/ text.

I tried to just make the text bold but in my local preview website it didn't make the difference i was expecting hence h5.

Any alternative ?

Reply:
I completely agree that those pillars should be differentiated. Don't H3s work here? 

I prefer semantic markup for hierarchy over appearance. There's also the accessibility consideration: https://www.w3.org/WAI/tutorials/page-structure/headings/

> Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a <h2> is not followed directly by an <h4>, for example.

Reply:
ack, will give it a go and then you will see if it makes sense with h3 as part of the review

Reply:
@cody-clark please help review again, made the change from h5 to h3